README.md

```diff
@@ -718,7 +718,12 @@ considered to be well-supported.
 
 To run the ORT binaries (also see [Installation from binaries](#from-binaries)) at least Java 11 is required. Memory and
 CPU requirements vary depending on the size and type of project(s) to analyze / scan, but the general recommendation is
-to configure Java with 8 GiB of memory (`-Xmx=8g`) and to use a CPU with at least 4 cores.
+to configure Java with 8 GiB of memory and to use a CPU with at least 4 cores.
+
+~~~bash
+# give Java Virtual Machine 8GB RAM
+export JAVA_OPTS="-Xmx8g"
```

Comment:
Note: This example is now Linux-specific and overrides any existing `JAVA_OPTS`. The former is probably ok, but the latter could be addressed with e.g. `export JAVA_OPTS="$JAVA_OPTS -Xmx8g"`.

Reply:
Is it okay, now?